dbt/adapters/bigquery/connections.py

```diff
@@ -323,17 +330,24 @@ def open(cls, connection):
         return connection
 
     @classmethod
-    def get_timeout(cls, conn):
+    def get_job_execution_timeout_seconds(cls, conn):
```

Comment:
There are some integration test failures like:
```
Unhandled error while executing ...
'BigQueryConnectionManager' object has no attribute 'get_timeout'
```

We need to replace `get_timeout` &rarr; `get_job_execution_timeout_seconds` within `load_dataframe`, to fix `dbt seed`:
https://github.com/dbt-labs/dbt-bigquery/blob/bc6113cc8efbc6a6f4618b5839e5a8873a48d2d2/dbt/adapters/bigquery/impl.py#L700

Plus one unit test:
https://github.com/dbt-labs/dbt-bigquery/blob/bc6113cc8efbc6a6f4618b5839e5a8873a48d2d2/tests/unit/test_bigquery_adapter.py#L603

Reply:
I could look into that